improver/utilities/time_lagging.py

```diff
@@ -36,6 +37,18 @@ def process(self, cubelist: Union[List[Cube], CubeList]) -> Cube:
         Returns:
             Concatenated forecasts
         """
+        if len(cubelist) == 1:
```

Comment:
Not a required change, but I typically would you `as_cubelist(cubes)` and adding *cubes to the args of the function.  This makes it a bit more flexible to how one calls and works with the plugin.